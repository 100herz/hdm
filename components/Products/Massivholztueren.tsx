import styled from "styled-components"

const Grid = styled.div`
  display: grid;
  grid-template-areas:
    "grid-item-1"
    "grid-item-2"
    "grid-item-3";
  grid-template-columns: 1fr;

  @media (min-width: 768px) {
    grid-template-areas:
      "grid-item-1 grid-item-2"
      "grid-item-3 grid-item-2";
    grid-template-columns: 1fr 1fr;
    column-gap: 8rem;
  }

  & > :nth-child(1) {
    grid-area: grid-item-1;
  }

  & > :nth-child(2) {
    grid-area: grid-item-2;
  }

  & > :nth-child(3) {
    grid-area: grid-item-3;
  }
`

type Props = {
  className?: string
}

const Massivholztueren: React.FC<Props> = ({ className = "" }: Props) => {
  return (
    <section className={`overflow-x-hidden relative lg:mt-24 ${className}`}>
      <Grid className="container pt-20 pb-20 bg-gray-100">
        <header className="lg:ml-32 lg:mt-8">
          <div>
            <h2 className="text-4xl font-bold lg:text-5xl">Massivholztüren</h2>
          </div>
        </header>

        <picture className="block mt-16 -mx-6 lg:ml-0 lg:-mr-32 lg:mt-0">
          <source srcSet="/images/hdm-massivholztueren.webp" type="image/webp" />
          <source srcSet="/images/hdm-massivholztueren.png" type="image/png" />
          <img
            src="/images/hdm-massivholztueren.png"
            alt="Massivholztüren - Sorglos die Schönheit massiven Holzes genießen"
          />
        </picture>

        <div className="lg:-mt-32 lg:ml-32">
          <p className="mt-12 lg:text-lg">
            Sorglos die Schönheit massiven Holzes genießen. Massivholz sorgt für behagliche Wärme und ist nie
            langweilig.
          </p>
          <p className="mt-8 lg:text-lg">
            Denn Holz ist pure Natur und bringt eine angenehme, freundliche Atmosphäre ins Haus, unterstrichen von
            ruhigen Licht- und Schattenspielen der offenen Lamellen- und Gittertüren.
          </p>

          <div className="mt-16 text-center lg:text-left lg:mt-24">
            <a
              href="/documents/hdm-cerastar.pdf"
              target="_blank"
              title="Den Katalog Cerastar ansehen und herunterladen"
              className="inline-block px-6 py-3 text-sm text-white rounded-full bg-hdm"
            >
              <svg
                className="inline-block mr-3"
                xmlns="http://www.w3.org/2000/svg"
                width="17.588"
                height="17.47"
                viewBox="0 0 17.588 17.47"
              >
                <path
                  d="M112.359,182.379a1.787,1.787,0,0,1-1.21-.524,2.06,2.06,0,0,1-.565-1.936,1.589,1.589,0,0,1,.686-1.008,19,19,0,0,1,4.113-2.3,44.427,44.427,0,0,0,2.419-5.041,11.514,11.514,0,0,1-1.29-3.75,3.57,3.57,0,0,1,.121-1.613c.4-1.29,1.29-1.331,1.653-1.29.081,0,1.492.2,1.734,2.218a10.9,10.9,0,0,1-.927,4.4,14,14,0,0,0,3.145,3.71,18.613,18.613,0,0,1,4.557.524,1.715,1.715,0,0,1,.927.565,1.5,1.5,0,0,1,.282,1.573h0a1.675,1.675,0,0,1-.323.565,1.558,1.558,0,0,1-1.976.444,21.456,21.456,0,0,1-3.871-2.54,16.963,16.963,0,0,0-5.565,1.169c-.686,1.25-1.532,2.621-2.5,4.153a1.7,1.7,0,0,1-.766.645h0C112.722,182.339,112.521,182.379,112.359,182.379Zm1.976-4.073a15.061,15.061,0,0,0-2.379,1.492c-.121.121-.242.242-.242.363a.956.956,0,0,0,.323.887.574.574,0,0,0,.484.121h0a.331.331,0,0,0,.2-.2C113.287,180.04,113.851,179.153,114.335,178.306Zm9.4-1.936a22.573,22.573,0,0,0,2.419,1.452.385.385,0,0,0,.484-.121,1.115,1.115,0,0,0,.161-.282c.081-.2.081-.282-.04-.4a.78.78,0,0,0-.363-.2A15.283,15.283,0,0,0,123.731,176.371Zm-5.242-3.589c-.4.927-.887,1.976-1.532,3.186a18.4,18.4,0,0,1,3.549-.726A20.147,20.147,0,0,1,118.489,172.782Zm-.4-6.775c-.161,0-.282.161-.4.524a2.408,2.408,0,0,0-.081,1.089,10.078,10.078,0,0,0,.686,2.339,8.044,8.044,0,0,0,.524-2.742c-.121-1.089-.686-1.21-.726-1.21Z"
                  transform="translate(-110.524 -164.909)"
                  fill="#fff"
                />
              </svg>
              Massivholztüren Katalog downloaden
            </a>
          </div>
        </div>
      </Grid>
    </section>
  )
}

export default Massivholztueren
