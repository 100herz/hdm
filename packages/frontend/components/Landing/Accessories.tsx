import Image from 'next/image'
import styled from '@emotion/styled'

export default function Accessories({ className = '' }: React.HTMLAttributes<HTMLElement>) {
  return (
    <section id="accessories" className={`bg-gray-100 overflow-x-hidden lg:mt-24 ${className}`}>
      <BackgroundImage>
        <div className="container py-20">
          <header className="lg:mt-16 lg:ml-32">
            <h2 className="text-4xl font-bold lg:text-5xl">
              DAZU<sup className="font-normal">&reg;</sup>
            </h2>
            <p className="mt-4 text-xl leading-snug lg:text-2xl">Das Zubehörprogramm</p>
          </header>

          <div className="lg:justify-between lg:ml-32 lg:flex lg:mt-8">
            <div className="lg:w-1/3 lg:-mt-4 lg:mr-12">
              <div className="mt-12 lg:-mx-24">
                <Image
                  src="/images/hdm-dazu-zubehoer.jpg"
                  alt="Möbelbauplatten - modernen Regalböden und Möbelbauplatten"
                  width={768}
                  height={432}
                />
              </div>
            </div>

            <div className="lg:w-1/2">
              <p className="mt-12 lg:text-lg">
                Alles was zum Anbringen von Dekorpaneele, zur Bodenverlegung, zum Bau von Mobiliar oder dem dekorativen
                Innenausbau notwendig ist, hat Holz Design Moers im DAZU<sup>&reg;</sup>-Zubehörprogramm.
              </p>
              <p className="mt-12 lg:text-lg">
                Verlegung und Montage einfach gemacht - mit dem DAZU<sup>&reg;</sup>-Zubehörprogramm.
              </p>
            </div>
          </div>
        </div>
      </BackgroundImage>
    </section>
  )
}

const BackgroundImage = styled.div`
  @media (min-width: 1024px) {
    background-image: linear-gradient(rgba(247, 250, 252, 0.95), rgba(247, 250, 252, 0.95)), url(/ci/hdm-bg.svg);
    background-repeat: no-repeat;
    background-position: left center;
    background-size: 100% auto;
  }
`
