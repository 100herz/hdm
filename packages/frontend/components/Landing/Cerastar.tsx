import styled from '@emotion/styled'
import dynamic from 'next/dynamic'

export default function Cerastar({ className = '' }: React.HTMLAttributes<HTMLElement>) {
  const Pdf = dynamic(() => import('@/components/Utils/Pdf'), { ssr: false })

  return (
    <section
      id="cerastar"
      className={`container relative lg:grid lg:grid-cols-2 lg:gap-32 mt-32 pt-20 pb-24 lg:mt-24 ${className}`}
    >
      <BackgroundImage />

      <header>
        <div className="lg:ml-32">
          <h2 className="text-4xl font-bold lg:text-5xl">Cerastar</h2>
          <p className="mt-4 text-xl leading-snug lg:text-2xl">
            Der neue Star unter <br />
            den Bodenbelägen
          </p>
        </div>

        <picture className="block mt-16 -mx-6 lg:-ml-20 lg:-mr-8">
          <source srcSet="/images/hdm-cerastar.webp" type="image/webp" />
          <source srcSet="/images/hdm-cerastar.png" type="image/png" />
          <img src="/images/hdm-cerastar.png" alt="Cerastar - der neue Star unten den Bodenbelägen" />
        </picture>
      </header>

      <div className="lg:mt-32 lg:pr-16">
        <h3 className="mt-12 font-bold lg:text-lg">
          Cerastar heißt der neue Designbodenbelag mit keramischer Nanocoat Oberflächenversiegelung.
        </h3>
        <p className="mt-8 lg:text-lg">
          1,2 Mio. feinste Keramikpartikel pro Quadratzentimeter machen die Oberfläche von Cerastar – gegenüber
          Oberflächen vergleichbarer Bodenbeläge – bis zu dreimal kratzfester und strapazierfähiger.
        </p>
        <p className="mt-8 lg:text-lg">
          So entsteht eine äußerst abriebfeste Schutzschicht, die deutlich resistenter gegen Mikrokratzer ist. Zugleich
          verleiht die hochtransparente Versiegelung den Dekoren eine einzigartige brillante Farbtiefe und eleganten
          Glanz.
        </p>
        <p className="mt-8 lg:text-lg">
          Weitere Informationen finden Sie unter{' '}
          <a className="underline text-hdm hover:text-hdm-dark" href="https://cerastar.eu">
            cerastar.eu
          </a>
          .
        </p>

        <div className="mt-16 text-center lg:text-left lg:mt-24">
          <Pdf title="Cerastar Katalog" filename="/documents/hdm-cerastar.pdf" />
        </div>
      </div>
    </section>
  )
}

const BackgroundImage = styled.div`
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: -2;
  overflow-x: hidden;
  background-color: #f8efef;

  &::before {
    position: absolute;
    top: 3rem;
    left: 45%;
    z-index: -1;
    width: 581px;
    height: 409px;
    background: url(/images/hdm-cerastar-muster.png) no-repeat;
    opacity: 0.7;
    content: '';
  }

  @media (min-width: 1024px) {
    &::before {
      top: 5rem;
    }
  }
`
