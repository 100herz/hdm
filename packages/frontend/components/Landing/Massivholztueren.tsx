import dynamic from 'next/dynamic'
// import Image from 'next/image'
import styled from '@emotion/styled'

export default function Massivholztueren({ className = '' }: React.HTMLAttributes<HTMLElement>) {
  const Pdf = dynamic(() => import('@/components/Utils/Pdf'), { ssr: false })

  return (
    <section id="massivholztueren" className={`overflow-x-hidden relative lg:mt-24 ${className}`}>
      <Grid className="container pt-20 pb-20 bg-gray-100">
        <header className="lg:ml-32 lg:mt-8 first">
          <h2 className="text-4xl font-bold lg:text-5xl">Massivholztüren</h2>
          <p className="mt-4 text-xl leading-snug lg:text-2xl">
            Das Lamellen- und <br /> Gittertüren Programm
          </p>
        </header>

        {/* TODO: `Image` is currently not available in SSG */}
        {/* <div className="mt-16 -mx-6 lg:ml-0 lg:-mr-32 lg:mt-0 second">
          <Image
            src="/images/hdm-massivholztueren.png"
            alt="Massivholztüren - Sorglos die Schönheit massiven Holzes genießen"
            width={768}
            height={779}
          />
        </div> */}
        <picture className="block mt-16 -mx-6 lg:ml-0 lg:-mr-32 lg:mt-0 second">
          <source srcSet="/images/hdm-massivholztueren.webp" type="image/webp" />
          <source srcSet="/images/hdm-massivholztueren.png" type="image/png" />
          <img
            src="/images/hdm-massivholztueren.png"
            alt="Massivholztüren - Sorglos die Schönheit massiven Holzes genießen"
            loading="lazy"
          />
        </picture>

        <div className="lg:-mt-16 lg:ml-32 third">
          <p className="mt-12 lg:text-lg">
            Sorglos die Schönheit massiven Holzes genießen. Denn Holz ist pure Natur und bringt eine angenehme,
            freundliche Atmosphäre ins Haus, unterstrichen von ruhigen Licht- und Schattenspielen der offenen Lamellen-
            und Gittertüren.
          </p>

          <p className="mt-8 lg:text-lg">Massivholz sorgt für behagliche Wärme und ist nie langweilig.</p>

          <div className="mt-16 text-center lg:text-left lg:mt-24">
            <Pdf title="Massivholztüren Katalog" filename="/documents/hdm-massivholztueren.pdf" />
          </div>
        </div>
      </Grid>
    </section>
  )
}

const Grid = styled.div`
  display: grid;
  grid-template-areas:
    'grid-item-1'
    'grid-item-2'
    'grid-item-3';
  grid-template-columns: 1fr;

  @media (min-width: 1024px) {
    grid-template-areas:
      'grid-item-1 grid-item-2'
      'grid-item-3 grid-item-2';
    grid-template-columns: 1fr 1fr;
    column-gap: 8rem;
  }

  & > .first {
    grid-area: grid-item-1;
  }

  & > .second {
    grid-area: grid-item-2;
  }

  & > .third {
    grid-area: grid-item-3;
  }
`
