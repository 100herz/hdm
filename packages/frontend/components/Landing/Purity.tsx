import dynamic from 'next/dynamic'
// import Image from 'next/image'
import styled from '@emotion/styled'

import PlayIcon from '@/components/UI/Icons/PlayIcon'
import ModalVideo from '@/components/UI/ModalVideo'
import { useVideo } from '@/hooks/use-video'

export default function Purity({ className = '' }: React.HTMLAttributes<HTMLElement>) {
  const Pdf = dynamic(() => import('@/components/Utils/Pdf'), { ssr: false })

  const [isModalOpen, handleModalOpen, handleModalClose, handleVideoClick] = useVideo()

  return (
    <section id="purity" className={`lg:relative overflow-x-hidden mt-12 lg:mt-32 pt-20 lg:pb-20 ${className}`}>
      <BackgroundImage />
      <header className="container mt-6">
        <div className="lg:ml-32">
          <h2 className="text-4xl font-bold lg:text-5xl">Purity</h2>
          <p className="mt-4 text-xl leading-snug lg:text-2xl">
            Laminatboden <br /> Bio Selection
          </p>
        </div>
      </header>

      <GridBio className="lg:container">
        <div className="relative mt-10 ml-6 lg:mt-16 lg:-ml-6 first">
          {/* TODO: `Image` is currently not available in SSG */}
          {/* <Image
            src="/images/hdm-purity-bio-selection.png"
            alt="Purity - Natürlich, sauber, rein"
            width={768}
            height={900}
          />
          <Badge className="hidden lg:block">
            <Image
              src="/certificates/hdm-oekosiegel.png"
              alt="HDM - Verantwortung für Mensch und Natur"
              width={231}
              height={231}
            />
          </Badge> */}
          <picture>
            <source srcSet="/images/hdm-purity-bio-selection.webp" type="image/webp" />
            <source srcSet="/images/hdm-purity-bio-selection.png" type="image/png" />
            <img src="/images/hdm-purity-bio-selection.png" alt="Purity - Natürlich, sauber, rein" loading="lazy" />
          </picture>
          <Badge
            className="hidden lg:block"
            src="/certificates/hdm-oekosiegel.png"
            alt="HDM - Verantwortung für Mensch und Natur"
          />
        </div>
        <div className="container lg:-mt-4 lg:px-0 second">
          <h3 className="text-xl font-bold">Bio ... Logisch!</h3>

          <p className="mt-8 lg:text-lg">
            Natürlich, sauber, rein – Bei den hochwertigen Purity Laminatböden der Bio-Selection ist der Name Programm.
            Rund 96&nbsp;% FSC-zertifiziertes Holz, die faktisch formaldehydfreie E0-Qualität und die Schadstofffreiheit
            dieser Böden sprechen für sich. Die Auszeichnung mit dem Blauen Engel bestätigt es.
          </p>

          <p className="mt-8 text-lg">
            Die einfache und schnelle Verlegung ist neben den idealen Pflegeeigenschaften ein weiteres Argument.
          </p>

          <div>
            <div>
              <h3 className="mt-8 font-bold ">Kollektion Glow – Böden mit Ausstrahlung</h3>
              <p>
                Eine deutlich sicht- und fühlbare Oberflächenstruktur kennzeichnet die Böden der Kollektion Glow. Der
                Boden gibt Inspiration für die Neugestaltung Ihrer Räume. Die umlaufende Fuge rundet das moderne
                Verlegebild ab.
              </p>
            </div>

            <div>
              <h3 className="mt-8 font-bold">Kollektion Pulse – Am Puls der Zeit</h3>
              <p>
                Authentische und fühlbare Maserungen folgen den Linien des Holzdekors – Dafür stehen die Laminatböden
                der Kollektion Pulse. Umlaufende Fugen geben dem Bild der verlegten Fläche Halt, Kontur und einen
                zeitgemäßen Stil.
              </p>
            </div>

            <div>
              <h3 className="mt-8 font-bold">Kollektion Stream - Alles andere als Mainstream</h3>
              <p>
                Ein Dielenformat von fast 25 cm Breite kennzeichnet diese Bigboards. Highlight ist die
                Oberflächenstruktur, die für außergewöhnliche Lichtbrechungen auf der Oberfläche sorgt. Kräftige
                Fugenlinien zeichnen das markante und individuelle Bild der verlegten Fläche.
              </p>
            </div>

            <p className="mt-8 font-semibold">
              Ein gutes Stück Natur: <br /> Laminatfußböden aus der Purity Bio Selection.
            </p>
          </div>
        </div>
        {/* TODO: `Image` is currently not available in SSG */}
        {/* <div className="container flex justify-between h-24 -mt-4 space-x-3 lg:space-x-6 lg:-mt-16 lg:-ml-6 lg:px-0 third">
          <Image src="/certificates/hdm-blauer-engel-176.png" alt="Blauer Engel 176" width={144} height={96} />
          <Image src="/certificates/hdm-fsc.png" alt="FSC - zertifiziertes Holz" width={82} height={96} />
          <Image src="/certificates/hdm-formaldehydfrei.png" alt="ohne Formaldehyd" width={96} height={96} />
          <Image src="/certificates/hdm-e0.svg" alt="E0 - DIN EN 717-1" width={96} height={96} />
        </div> */}
        <div className="container flex justify-between -mt-4 lg:space-x-6 lg:-mt-16 lg:-ml-6 lg:px-0 third">
          <img
            className="hidden h-24 lg:inline-block"
            src="/certificates/hdm-blauer-engel-176.png"
            alt="Blauer Engel 176"
            loading="lazy"
          />
          <img
            className="h-20 lg:h-24"
            src="/certificates/hdm-fsc.png"
            alt="FSC - zertifiziertes Holz"
            loading="lazy"
          />
          <img
            className="h-20 lg:h-24"
            src="/certificates/hdm-formaldehydfrei.png"
            alt="ohne Formaldehyd"
            loading="lazy"
          />
          <img className="h-20 lg:h-24" src="/certificates/hdm-e0.svg" alt="E0 - DIN EN 717-1" loading="lazy" />
        </div>
        <div className="container flex flex-col self-end text-center sm:flex-row lg:text-left lg:-mt-6 lg:px-0 fourth">
          <Pdf title="Purity Bio Selection" filename="/documents/hdm-purity-bio-selection.pdf" />
          <button
            className="inline-block px-6 py-3 mt-4 text-sm text-white rounded-full sm:mt-0 sm:ml-10 bg-hdm hover:bg-hdm-dark"
            onClick={handleModalOpen}
          >
            <PlayIcon /> Video ansehen
          </button>
          <ModalVideo
            videoName="hdm-purity"
            isOpen={isModalOpen}
            onClose={handleModalClose}
            onVideoClick={handleVideoClick}
          />
        </div>
      </GridBio>

      <div className="mt-40">
        <header className="container mt-6">
          <div className="lg:ml-32">
            <h2 className="text-4xl font-bold lg:text-5xl">Purity</h2>
            <p className="mt-4 text-xl leading-snug lg:text-2xl">
              Laminatboden <br /> Home Selection
            </p>
          </div>
        </header>

        <GridHome className="lg:container lg:mb-12">
          {/* TODO: `Image` is currently not available in SSG */}
          {/* <div className="mt-10 ml-6 lg:-mt-16 first">
            <Image
              src="/images/hdm-purity-home-selection.png"
              alt="Purity - Natürlich, sauber, rein"
              width={768}
              height={900}
            />
          </div> */}
          <picture className="mt-10 ml-6 lg:-mt-16 first">
            <source srcSet="/images/hdm-purity-home-selection.webp" type="image/webp" />
            <source srcSet="/images/hdm-purity-home-selection.png" type="image/png" />
            <img src="/images/hdm-purity-home-selection.png" alt="Purity - Natürlich, sauber, rein" />
          </picture>
          <div className="container mt-24 lg:grid lg:mt-20 second">
            <p className="lg:text-lg">
              In der Home-Selection der Purity-Laminatbodenbeläge by HDM zeichnet sich BLOOM durch beeindruckende
              Dekorbilder und eine feine Glattkante aus. <br /> <br /> Die Schadstofffreiheit dieser Böden spricht für
              sich, belegt durch die Auszeichnung mit dem Blauen Engel. Die einfache und schnelle Verlegung ist neben
              den idealen Pflegeeigenschaften ein weiteres Argument.
            </p>
            <h3 className="mt-8 text-xl font-bold">BLOOM – Einfach natürlich</h3>
          </div>
          {/* TODO: `Image` is currently not available in SSG */}
          {/* <div className="container flex justify-between h-24 mt-12 lg:mt-16 lg:space-x-6 third">
            <Image src="/certificates/hdm-blauer-engel-176.png" alt="Blauer Engel 176" width={144} height={96} />
            <Image src="/certificates/hdm-fsc.png" alt="FSC - zertifiziertes Holz" width={82} height={96} />
            <Image src="/certificates/hdm-e1.svg" alt="E1 - DIN EN 16156" width={96} height={96} />
          </div> */}
          <div className="container flex justify-between mt-12 lg:mt-16 lg:space-x-6 third">
            <img
              className="h-20 lg:h-24"
              src="/certificates/hdm-blauer-engel-176.png"
              alt="Blauer Engel 176"
              loading="lazy"
            />
            <img
              className="h-20 lg:h-24"
              src="/certificates/hdm-fsc.png"
              alt="FSC - zertifiziertes Holz"
              loading="lazy"
            />
            <img className="h-20 lg:h-24" src="/certificates/hdm-e1.svg" alt="E1 - DIN EN 16156" loading="lazy" />
          </div>
          <div className="container flex flex-col mt-16 text-center sm:flex-row lg:text-left lg:mt-24 fourth">
            <Pdf title="Purity Home Selection" filename="/documents/hdm-purity-home-selection.pdf" />
            <button
              className="inline-block px-6 py-3 mt-4 text-sm text-white rounded-full sm:mt-0 sm:ml-10 bg-hdm hover:bg-hdm-dark"
              onClick={handleModalOpen}
            >
              <PlayIcon /> Video ansehen
            </button>
            <ModalVideo
              videoName="hdm-purity"
              isOpen={isModalOpen}
              onClose={handleModalClose}
              onVideoClick={handleVideoClick}
            />
          </div>
        </GridHome>
      </div>
    </section>
  )
}

const BackgroundImage = styled.div`
  @media (min-width: 1024px) {
    position: absolute;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    z-index: -2;
    background-color: #f7fafc;

    &::before {
      position: absolute;
      top: 5%;
      right: -50%;
      z-index: -1;
      width: calc(2.9333 * 90%);
      height: 90%;
      background: url(/ci/hdm-bg.svg) no-repeat;
      opacity: 0.05;
      content: '';
    }
  }
`

const GridBio = styled.div`
  display: grid;
  grid-template-areas:
    'grid-item-1'
    'grid-item-2'
    'grid-item-3'
    'grid-item-4';
  grid-template-columns: 1fr;
  gap: 6rem;

  @media (min-width: 1024px) {
    grid-template-areas:
      'grid-item-1 grid-item-2'
      'grid-item-3 grid-item-4';
    grid-template-columns: 1.1fr 0.9fr;
  }

  & > .first {
    grid-area: grid-item-1;
  }

  & > .second {
    grid-area: grid-item-2;
  }

  & > .third {
    grid-area: grid-item-3;
  }

  & > .fourth {
    grid-area: grid-item-4;
  }
`

const GridHome = styled.div`
  display: grid;
  grid-template-areas:
    'grid-item-1'
    'grid-item-2'
    'grid-item-3'
    'grid-item-4';
  grid-template-columns: 1fr;
  column-gap: 6rem;

  @media (min-width: 1024px) {
    grid-template-areas:
      'grid-item-2 grid-item-1'
      'grid-item-3 grid-item-1'
      'grid-item-4 grid-item-1';
    grid-template-columns: 0.9fr 1.1fr;
  }

  & > .first {
    grid-area: grid-item-1;
  }

  & > .second {
    grid-area: grid-item-2;
  }

  & > .third {
    grid-area: grid-item-3;
  }

  & > .fourth {
    grid-area: grid-item-4;
    align-self: flex-end;
  }
`

// const Badge = styled.div`
const Badge = styled.img`
  position: absolute;
  bottom: 5rem;
  left: 5rem;
`
