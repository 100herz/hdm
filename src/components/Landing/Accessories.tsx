import styled from "styled-components"

const Grid = styled.div`
  display: grid;
  grid-template-areas:
    "grid-item-1"
    "grid-item-2";
  grid-template-columns: 1fr;

  @media (min-width: 768px) {
    grid-template-areas: "grid-item-1 grid-item-2";
    grid-template-columns: 1fr 1fr;
    column-gap: 8rem;
  }

  & > :nth-child(1) {
    grid-area: grid-item-1;
  }

  & > :nth-child(2) {
    grid-area: grid-item-2;
  }
`

const BackgroundImage = styled.div`
  @media (min-width: 768px) {
    background-image: linear-gradient(rgba(247, 250, 252, 0.95), rgba(247, 250, 252, 0.95)), url(/ci/hdm-bg.svg);
    background-repeat: no-repeat;
    background-position: left bottom;
    background-size: 100% auto;
  }
`

type Props = {
  className?: string
}

const Accessories: React.FC<Props> = ({ className = "" }: Props) => {
  return (
    <section id="accessories" className={`lg:bg-gray-100 overflow-x-hidden lg:mt-24 ${className}`}>
      <BackgroundImage>
        <Grid className="container pt-20 lg:pb-20">
          <div className="lg:max-w-md">
            <header className="mt-16 lg:ml-32">
              <h2 className="text-4xl font-bold leading-tight lg:text-5xl">Regalböden / Möbelbauplatten</h2>
            </header>

            <picture className="block mt-12 lg:-mx-24">
              <source srcSet="/images/hdm-moebelbauplatten.webp" type="image/webp" />
              <source srcSet="/images/hdm-moebelbauplatten.jpg" type="image/jpeg" />
              <img
                src="/images/hdm-moebelbauplatten.webp"
                alt="Möbelbauplatten - modernen Regalböden und Möbelbauplatten"
              />
            </picture>

            <div className="">
              <p className="mt-12 lg:text-lg">
                Mit modernen Regalböden und Möbelbauplatten, in vielen Abmessungen und dem richtigen Zubehör, lässt sich
                einfach und schnell frischer Wind in alle Räume bringen.
              </p>
              <p className="mt-8 lg:text-lg">
                Ihrer Kreativität können Sie mit dem Holzzuschnitt-Programm von Holz Design Moers freien Lauf lassen.
              </p>
            </div>
          </div>

          <div className="lg:max-w-md">
            <header className="mt-16 lg:ml-32">
              <h2 className="text-4xl font-bold lg:text-5xl">
                DAZU<sup className="font-normal">&reg;</sup>
              </h2>
              <p className="mt-4 text-xl leading-snug lg:text-2xl">Das Zubehörprogramm</p>
            </header>

            <picture className="block mt-12 lg:-mx-24">
              <source srcSet="/images/hdm-dazu-zubehoer.webp" type="image/webp" />
              <source srcSet="/images/hdm-dazu-zubehoer.webp" type="image/jpeg" />
              <img src="/images/hdm-dazu-zubehoer.webp" alt="DAZU-Zubehörprogramm" />
            </picture>

            <div className="">
              <p className="mt-12 lg:text-lg">
                Alles was zum Anbringen von Dekorpaneele, zur Bodenverlegung, zum Bau von Mobiliar oder dem dekorativen
                Innenausbau notwendig ist, hat Holz Design Moers im DAZU<sup>&reg;</sup>-Zubehörprogramm. Verlegung und
                Montage einfach gemacht - mit dem DAZU<sup>&reg;</sup>-Zubehörprogramm.
              </p>
            </div>
          </div>
        </Grid>
      </BackgroundImage>
    </section>
  )
}

export default Accessories
