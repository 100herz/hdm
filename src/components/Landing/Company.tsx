import styled from "styled-components"

const Grid = styled.div`
  display: grid;
  grid-template-areas:
    "grid-item-1"
    "grid-item-2";
  grid-template-columns: 1fr;

  @media (min-width: 1024px) {
    grid-template-areas: "grid-item-1 grid-item-2";
    grid-template-columns: 1fr 1fr;
    column-gap: 4rem;
  }

  & > :nth-child(1) {
    grid-area: grid-item-1;
  }

  & > :nth-child(2) {
    grid-area: grid-item-2;
  }
`

type Props = {
  className?: string
}

const Company: React.FC<Props> = ({ className = "" }: Props) => {
  return (
    <section
      className={`bg-gray-100 lg:bg-transparent mt-10 pt-10 lg:max-w-1400 mx-auto overflow-x-hidden lg:mt-24 ${className}`}
    >
      <header className="container lg:mt-16 lg:ml-24">
        <h2 className="text-4xl font-bold lg:text-5xl">Unternehmen</h2>
      </header>

      <div className="relative mt-12 lg:mt-16 pb-16/9">
        <iframe
          className="absolute w-full h-full"
          src="https://www.youtube-nocookie.com/embed/-7ki7fK1prA?rel=0"
          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture; fullscreen"
        ></iframe>
      </div>

      <div className="lg:bg-gray-100">
        <Grid className="container pt-10 pb-20 lg:pt-20">
          <div>
            <h3 className="font-bold text-hdm lg:text-xl">
              Holz Design Moers GmbH ist strategischer Partner für Baumarktgruppen, Baufachmärkte und den Fachhandel.
            </h3>

            <p className="mt-8 lg:mt-16 lg:text-lg lg:ml-24">
              Als Vollsortimenter für den Innenausbau bietet das Unternehmen seinen Handelspartnern ein umfangreiches
              Portfolio zur Verkleidung und Gestaltung von Boden, Wand und Decke: Laminat- und Designböden, Wand- und
              Deckenpaneele, dazu ein großes Massivholz- und Holzwerkstoffsortiment: Leisten sowie Regalböden und
              Möbelbauplatten.
            </p>
            <p className="mt-8 lg:text-lg lg:ml-24">
              Das Angebot wird durch ein umfassendes Zubehörprogramm ergänzt und komplettiert.
            </p>
          </div>

          <div className="mt-8 lg:mt-0">
            <p className="lg:text-lg lg:mx-12">
              Bereits <strong className="text-hdm">1968</strong> präsentierte Firmengründer Arnold Dammers in Moers das
              weltweit erste Dekorpaneel.
            </p>
            <p className="mt-8 lg:text-lg lg:mx-12">
              <strong className="text-hdm">1981</strong> entwickelte Dammers die dekorativ ummantelte Leiste, die sich
              heute als UMA®-Leiste im Markt etabliert hat. HDM gilt als Erfinder der beiden trendsetzenden Produkte für
              den dekorativen Innenausbau.
            </p>
            <p className="mt-8 lg:text-lg lg:mx-12">
              <strong className="text-hdm">1989</strong> folgte die Entwicklung einer Elektronenstrahlgehärteten
              Flächenveredlung, die unter dem Markennamen ELESGO® eingeführt wurde. Die besonders kratzfeste Oberfläche
              veredelt unter anderem Labortische, Möbelfronten und liefert die Ideen für viele weitere Anwendungen.
            </p>
            <p className="mt-8 lg:text-lg lg:mx-12">
              Im Jahr <strong className="text-hdm">2000</strong> lancierte das Moerser Unternehmen den ersten
              Superglanz- Laminatboden mit Elesgo-Oberfläche.
            </p>
          </div>
        </Grid>
      </div>
    </section>
  )
}

export default Company
