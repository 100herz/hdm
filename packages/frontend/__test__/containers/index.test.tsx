import Accessories from '@/components/Landing/Accessories'
import Cerastar from '@/components/Landing/Cerastar'
import Company from '@/components/Landing/Company'
import Crossflex from '@/components/Landing/Crossflex'
import Massivholztueren from '@/components/Landing/Massivholztueren'
import Purity from '@/components/Landing/Purity'
import IndexPage from '@/containers/index'
import { shallow, ShallowWrapper } from 'enzyme'
import toJSON from 'enzyme-to-json'

describe('IndexPage Testings', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<IndexPage />)
  })

  test('matches snapshot', () => {
    expect(toJSON(wrapper)).toMatchSnapshot()
  })

  test('render one Crossflex component', () => {
    expect(wrapper.find(Crossflex)).toHaveLength(1)
  })

  test('render one Purity component', () => {
    expect(wrapper.find(Purity)).toHaveLength(1)
  })

  test('render one Cerastar component', () => {
    expect(wrapper.find(Cerastar)).toHaveLength(1)
  })

  test('render one Massivholztueren component', () => {
    expect(wrapper.find(Massivholztueren)).toHaveLength(1)
  })

  test('render one Accessories component', () => {
    expect(wrapper.find(Accessories)).toHaveLength(1)
  })

  test('render one Company component', () => {
    expect(wrapper.find(Company)).toHaveLength(1)
  })
})
