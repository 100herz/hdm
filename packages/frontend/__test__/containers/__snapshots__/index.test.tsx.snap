// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage Testings matches snapshot 1`] = `
<Layout
  title="HDM â€“ Holz Design Moers"
>
  <div
    id="produkte"
  >
    <Crossflex />
    <Purity />
    <Cerastar />
    <Massivholztueren />
    <Shelves />
    <Strips />
    <Accessories />
  </div>
  <div
    id="unternehmen"
  >
    <Company />
  </div>
</Layout>
`;
